packages/terra-menu/src/Menu.jsx

```diff
@@ -108,6 +192,18 @@ class Menu extends React.Component {
     });
   }
 
+  handleKeydown(e) {
```

Comment:
Do we need this for menu..? Correct me if am wrong because menu has popup which uses hookshot-content and this is being handled in hookshot-content so I don't think we need to handle here.

Reply:
Removed in 15a749c